### PR TITLE
Add Angle Marks

### PR DESCRIPTION
**Pull request by dormaayan**
This PR solves issue #64.
I added a new shape to Style named `AngleMark`
`AngleMark` allows you specify whether this is a **right angle** or a **regular angle** by specifying the parameter `isRight = "true"\"false"`. Other parameters you might specify for the regular angle sign are the radius, the rotation of the angle and the angle itself.
I added prependicular mark to the following .sty files:

- surjection_computed.sty:
image

- cart-test-pt.sty:
image

I also added my own examples, both run with angles.sub:

- regularAngle.sty (these are 2 different angles):
image
image

- rightAngle.sty:
image

@hypotext , @wodeni 
The system pass the tests after those changes, please let me know if you have comments

### CODE REVIEW

File: src/Shapes.hs [hunk 1]
```diff
@@ -197,6 +197,36 @@ instance Named Square where
...
+-------------------------- (Angle Mark, added by Dor)
+data AngleMark = AngleMark { xam :: Float -- Starting point for angle mark
+                     , yam :: Float
+                     , radiusam :: Float -- The radius of the angle mark
+                     , sizeam :: Float
+                     , nameam :: String
+                     , coloram :: Color
+                     , angleam :: Float -- The angle that the angle mark specifies
+                     , rotationam :: Float -- The rotation angle of amgle mark
+                     , isRightam :: String -- Is this a right angle or not
```
Comment (hypotext) [inline src/Shapes.hs]: This is probably better as a bool or an enum `data AngleType = Right | NotRtAngle`

Comment (dormaayan) [reply src/Shapes.hs]: Yes I know, I had trouble doing that at the begging but now I think I know how to solve that

File: src/sty/regularAngle.sty [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+Set `B` {
+  shape = Arrow {
+    start = origin
+    end   = vert
+    text = None
+  }
+
+    angleMark2 = AngleMark{
```
Comment (hypotext) [inline src/sty/regularAngle.sty]: Style style guideline: space between keyword and brace

File: src/sty/surjection_computed.sty [hunk 3]
```diff
@@ -21,7 +21,12 @@ Set `codomain` {
           text = "f(x)"
     }
 
-    objective nearHead(codomain, codomain.shape.label, 0, 20) -- TODO parse negative numbers
+    angleNotation = AngleMark {
+      start = atOrigin()
```
Comment (hypotext) [inline src/sty/surjection_computed.sty]: how are the other parameters of the arc being determined here?

**Comment by hypotext** [comment]
It would also be nice to be able to choose from two arc styles "line" or "wedge," the latter of which fills in the angle with a solid color of lower opacity, as in Keenan's blog post on complex numbers.

**Comment by hypotext** [comment]
Thanks for the changes so far! Let me know when the `from` and `to` API is ready, and I'll give the PR a second review. Also, it would be nice to be able to turn off the arc outline if there is a fill, but that's not super important.

BTW, Keenan requests that all fills be done with the color `#1b1f8a` at opacity 20%. :-)

**Pull request merged**